source/backend/api/Services/NoteService.cs

```diff
@@ -18,116 +14,134 @@ namespace Pims.Api.Services
     public class NoteService : BaseService, INoteService
     {
         private readonly INoteRepository _noteRepository;
-        private readonly IEntityNoteRepository _entityNoteRepository;
-        private readonly IMapper _mapper;
+        private readonly INoteRelationshipRepository<PimsAcquisitionFileNote> _acquisitionFileNoteRepository;
+        private readonly INoteRelationshipRepository<PimsDispositionFileNote> _dispositionFileNoteRepository;
+        private readonly INoteRelationshipRepository<PimsLeaseNote> _leaseNoteRepository;
+        private readonly INoteRelationshipRepository<PimsManagementFileNote> _managementFileNoteRepository;
+        private readonly INoteRelationshipRepository<PimsProjectNote> _projectNoteRepository;
+        private readonly INoteRelationshipRepository<PimsPropertyNote> _propertyNoteRepository;
+        private readonly INoteRelationshipRepository<PimsResearchFileNote> _researchFileNoteRepository;
```

Comment:
Replaced **EntityNoteRepository** (which was growing and growing) with individual repos for each type of note-relationship (project-note, acquisitionfile-note, etc)

This is consistent with the work Manuel did on the Property Documents refactor